src/Symfony/Component/ExpressionLanguage/ParserCache/ArrayParserCache.php

```diff
@@ -15,6 +15,8 @@
 
 /**
  * @author Adrien Brault <adrien.brault@gmail.com>
+ *
+ * @deprecated ParserCacheInterface is deprecated. Use Symfony\Component\Cache\Adapter\ArrayAdapter instead
```

Comment:
I'm sorry I've not found any example of deprecation triggering below namespace declaration, could you show me please ?
I'm not sure to understand your second sentence : there is no need to add @deprecated tag on ParserCacheInterface.php ?